### PR TITLE
updates

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: PRS1001-2000/PRS1016AbarraG.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <persName type="alt">ʾAbbā Sǝbbār</persName>
                <nationality type="Ethiopia post-Aksumite">Christian Ethiopia</nationality>
                <faith type="EOTC">EOTC</faith>
-               <occupation>daǧǧ</occupation>
+               <persName><roleName type="title">daǧǧāzmāč</roleName></persName>
```
Comment (eu-genia) [inline PRS1001-2000/PRS1016AbarraG.xml:55]: i would say occupation type military for dajjazmach actually...

File: PRS1001-2000/PRS1017AbarraK.xml [hunk 2]
```diff
@@ -51,7 +51,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                </persName>
                <nationality type="Ethiopia post-Aksumite">Christian Ethiopia</nationality>
                <faith type="EOTC">EOTC</faith>
-               <occupation>däǧǧ. </occupation>
```
Comment (eu-genia) [inline PRS1001-2000/PRS1017AbarraK.xml:54]: `<occupation type="military">daǧǧāzmāč</occupation>`?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: PRS1001-2000/PRS1016AbarraG.xml (hunk 1 above)
Comment (thea-m) [reply PRS1001-2000/PRS1016AbarraG.xml:55]: I don't know. I don't know the history of these titles well enough. I had searched for daǧǧāzmāč and found that it was most often encoded as title (but occupation type military is also used).

Comment (eu-genia) [reply PRS1001-2000/PRS1016AbarraG.xml:55]: it is both, it is a title and a military function.

**Comment by thea-m** [comment]
There will also be a problem with `<occupation>sch.</occupation>`, which is the source of many invalid records. I would tend to batch replace it with type academic, but probably that would not be the best choice for each single record. But I'm not willing to go through the records one by one and research each person. What would you suggest?

**Comment by eu-genia** [comment]
I am perfectly fine with batch replacing to academic.

**Pull request merged**